flight4handler.go

```diff
@@ -318,15 +322,25 @@ func flight4Generate(c flightConn, state *State, cache *handshakeCache, cfg *han
 		})
 
 		if cfg.clientAuth > NoClientCert {
+			// An empty list of certificateAuthorities signals to
+			// the client that it may send any certificate in response
+			// to our request. When we know the CAs we trust, then
+			// we can send them down, so that the client can choose
+			// an appropriate certificate to give to us.
+			var certificateAuthorities [][]byte
+			if cfg.clientCAs != nil {
+				certificateAuthorities = cfg.clientCAs.Subjects()
```

Comment:
`SertPool.Subjects()` seems be marked as deprecated since Go 1.18.
https://pkg.go.dev/crypto/x509#CertPool.Subjects
https://tip.golang.org/doc/go1.18#crypto/x509
I haven't dug into the reason and alternatives.

Reply:
Yes. I know, but `CertPool` doesn't provide any alternatives. When I look to source code of go 1.18 they still use it:
https://cs.opensource.google/go/go/+/master:src/crypto/tls/handshake_server_tls13.go;l=593;drc=fe4de36198794c447fbd9d7cc2d7199a506c76a5

Reply:
Looks fine to keep using as long as crypto/tls is using too.